content/projects/github-api-consume-part-2/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+content_type: project
+flavours:
+- any_language
+learning_outcomes:
+- web_dev_http_requests
+- web_dev_api_call_tools
+- web_dev_restful_apis
+- web_dev_json_data_representation
+- web_dev_api_consumption
+prerequisites:
+  hard:
+  - projects/github-api-consume-part-1
+  soft: []
+ready: true
+story_points: 3
+submission_type: repo
+tags:
+- pagination
+title: Add pagination to Consume Github API
+---
+
+In part 1 of this project, you made use of Github's awesome API. Now that you are familiar with how it works, you are going to upgrade that project by adding pagination. 
+
+API Pagination is essential if you're dealing with alot of data and endpoints. It helps make your response easier to handle.
```

Comment:
This sentence isn't really useful. It would be better to tell people what pagination actually is or link to a resource that explains it.